src/polymer/polymer-element.ts

```diff
@@ -400,6 +408,10 @@ function applySelf(
   scannedElement.events.forEach((o) => element.events.push(o));
   element.extends = scannedElement.extends;
   element.jsdoc = scannedElement.jsdoc;
+  if (!element.jsdoc && element.description) {
+    element.jsdoc = jsdoc.parseJsdoc(element.description);
```

Comment:
This is no longer needed because the jsdoc demo tag processing takes place at the end of resolving elements now.